tutorials/dnssec-cloud-dns-domains/index.md

```diff
@@ -0,0 +1,1074 @@
+
+This tutorial is intended for DNS domain administrators using Google Cloud DNS
+who have enabled DNSSEC for their managed zones; it shows how to activate DNSSEC
+for those domains by adding DS records through their domain registrars. The
+specifics vary, depending on which domain registrar is used; while this tutorial
+does not provide instructions for all domain registrars, it does so (with links
+to online resources) for the most popular ones that support DNSSEC.
+
+This tutorial also provides instructions for *de*-activating DNSSEC by removing
+the DS record, an essential step that **must** be performed before disabling
+DNSSEC for Google Cloud DNS.
+
+## Objectives
+
+1.  Confirm DNSSEC support by domain registrar and TLD registry.
+
+2.  Confirm full propagation of DNSSEC records and get DNSKEY information.
+
+3.  Activate DNSSEC by adding a DS record through the domain registrar.
+
+4.  Confirm DNSSEC validation of the domain.
+
+Alternately, once you have already activated DNSSEC for a domain:
+
+1.  De-activate DNSSEC by removing the DS record through the domain registrar.
+
+2.  Confirm propagation of the DS removal by the TLD registry.
+
+**Figure 1.** *Overview of tutorial components*
+
+![Diagram of DNS delegation and registration](overview.png)
+
+## Before you begin
+
+This tutorial assumes that you
+
+-   already own (or are an administrator for) a domain,
+
+-   have access to a delegated Google Cloud DNS managed zone that is
+    authoritative for that domain, and
+
+-   have [enabled DNSSEC signing][3] for that managed zone.
+
+[3]: https://cloud.google.com/dns/dnssec
+
+You also need an online account with the domain registrar (or credentials to
+request them to add or remove DS records from the TLD registry).
+
+### Domain registrar account
+
+If you are unsure which domain registrar is responsible for a domain, you can
+use the `whois` command line tool (see below) or do a whois lookup at
+[GWhois.org][4] ([example][5]). The domain may have been registered through a
+reseller; check the WHOIS results for reseller information and if one is listed,
+check the domain registrar-specific instructions in this tutorial for both the
+domain registrar and the reseller.
+
+[4]: https://gwhois.org/
+[5]: https://gwhois.org/dns-example.info+dns
+
+~~~console
+$ whois dns-example.info
+Domain Name: DNS-EXAMPLE.INFO
+Registry Domain ID: D503300000000040442-LRMS
+Registrar WHOIS Server:
+Registrar URL: http://www.google.com
+…
+Registrar: Google Inc.
+Registrar IANA ID: 895
+…
+Reseller:
+…
+Name Server: NS-CLOUD-E1.GOOGLEDOMAINS.COM
+Name Server: NS-CLOUD-E2.GOOGLEDOMAINS.COM
+…
+~~~
+
+#### TLD *registries* vs. domain *registrars*
+
+Despite their similar names, TLD **registries** and domain **registrars** are
+separate organizations with different roles, although for .gov and .int TLDs,
+and some two-letter country code (ccTLD) top-level domains, the TLD registry may
+also provide domain registrar services.
+
+<dl>
```

Comment:
Any and all HTML-like content in the markdown files are stripped out when they are published, you'll have to remove the HTML in your tutorial.

Reply:
Done. As this is not mentioned anywhere in the [Style Guide](https://cloud.google.com/community/tutorials/styleguide#general-style-guidelines) should I submit a separate PR to add a "Markdown usage guidelines" section that notes this restriction, as well as the specific Markdown flavor (GFM)? Other things could be added to this section later, but these are likely to be issues for other submitters as well.

Reply:
Sure, we support plain markdown, no special flavor.

One thing of note: code fences don't work within lists.